phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/skin/ListWidgetSelectorSkin.java

```diff
@@ -0,0 +1,104 @@
+package org.phoenicis.javafx.components.skin;
+
+import javafx.scene.control.ToggleButton;
+import javafx.scene.control.ToggleGroup;
+import javafx.scene.layout.HBox;
+import org.phoenicis.javafx.components.behavior.ListWidgetSelectorBehavior;
+import org.phoenicis.javafx.components.control.ListWidgetSelector;
+import org.phoenicis.javafx.views.common.widgets.lists.ListWidgetType;
+
+/**
+ * The skin for the {@link ListWidgetSelector} component
+ */
+public class ListWidgetSelectorSkin
```

Comment:
Naming: Is it really a "skin" or rather a "ui"? For me, skins just look different while this can be implemented with completely different controls (if I understand it correctly).

Reply:
As I understand it, "skin" is the JavaFX name for "ui".
For more details take a look at the [Skin](https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/Skin.html) class, which is the upper most parent class of all our `Skin` classes.

Reply:
I wasn't aware of this but if it's the normal JavaFX naming, it's fine for me.